examples/get-started/scripting/carto/index.html

```diff
@@ -0,0 +1,135 @@
+<html>
+  <head>
+    <!-- deck.gl standalone bundle -->
+    <script src="https://unpkg.com/deck.gl@^8.1.0/dist.min.js"></script>
```

Comment:
You will need to include the carto module from CDN here.

As the number of submodules grow, we have paused adding new modules into the main bundle to prevent bloating its size with unnecessary functionalities, unless it can be justified for most common use cases. For example, the arcgis module is not part of the main bundle.

Reply:
Done. I've kept it linked against ^8.1.0 just to allow an easy replace 😄